example_site/src/components/Collections.js

```diff
@@ -124,23 +211,42 @@ export default function Collections() {
       ) : null}
 
       {isSuccess ? (
-        <>{/*
-          <h3>Template Filters</h3>
-        <JsonEditor value={templateFilter} onChange={setTemplateFilter} /> */}
+        <>
           <div className="browser-columns">
-            <div className="categories">
+            <div className="filters">
+              <h2>Duration</h2>
+              {durationFilters.map((filter) => (
+                <DurationFilter
+                  filter={filter}
+                  templateFilter={templateFilter}
+                  setTemplateFilter={setTemplateFilter}
+                />
+              ))}
+
+              <h2>Aspect Ratio</h2>
+              {aspectRatioFilters.map((filter) => (
+                <AspectRatioFilter
+                  filter={filter}
+                  templateFilter={templateFilter}
+                  setTemplateFilter={setTemplateFilter}
+                />
+              ))}
+
               <h2>Categories</h2>
               {collections.map((collection) => (
                 <CollectionNames
```

Comment:
While we're at it, let's rename this to `CollectionFilter` to stay consistent with the other components.